inventory/group_vars/hu.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+# See https://github.com/openfoodfoundation/ofn-install/wiki/Setup for more info
+checkout_zone: Europe
+country_code: HU
+default_country_code: HU
+currency: HUF
+locale: hu_HU                  # Default locale for OFN instance
```

Comment:
I believe this should be `hu` as well to conform with the[ `hu.yml` file ](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/config/locales/hu.yml)

Reply:
Changed to hu.